USAGE.md

```diff
@@ -502,6 +503,39 @@ However, limited testing has shown `dunst` will not complain if its configuratio
 hook = "cat ~/.config/dunst/dunstrc.local %f > ~/.config/dunst/dunstrc && systemctl --user restart dunst"
 ```
 
+## Delta
+
+Add the following to `~/.config/tinted-theming/tinty/config.toml`:
+
+```toml
+[[items]]
+path = "https://github.com/tinted-theming/tinted-delta"
+name = "delta"
+themes-dir = "configs"
+hook = "cp -f %f ~/.config/tinted-theming/delta.gitconfig"
```

Comment:
It's not necessary to copy the file to a new location in this case since we can just reference where the original file is located (in the next  suggestion in the gitconfig).

```suggestion
```